src/components/ReportActionsSkeletonView/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../withWindowDimensions';
+import SkeletonViewLines from './SkeletonViewLines';
+import CONST from '../../CONST';
+
+const propTypes = {
+    ...windowDimensionsPropTypes,
+};
+
+const ReportActionsSkeletonView = (props) => {
+    const possibleVisibleContentItems = Math.floor(props.windowHeight / CONST.CHAT_SKELETON_VIEW.AVERAGE_ROW_HEIGHT);
```

Comment:
Should this be based on the height of the parent `ReportScreen` instead of the window? This doesn't account for the height of the report header or the chat composer.

Reply:
Thanks for this. I couldn't make out why it was a bit off. Now I realize it was going behind the composer.  I've added the fix.